python/mirascope/llm/tools/tools.py

```diff
@@ -1,30 +1,26 @@
-"""The `ContextTool` class for defining tools that LLMs can request be called."""
+"""The `Tool` class for defining tools that LLMs can request be called."""
 
 from __future__ import annotations
 
 import json
 from collections.abc import Awaitable
-from typing import Any, Generic, Protocol, cast
-from typing_extensions import TypeVar
+from typing import Any, Generic, Protocol, TypeVar, cast
 
 from ..content import ToolCall, ToolOutput
 from ..context import Context, DepsT
 from ..types import AnyP, JsonableCovariantT
-from .protocols import AsyncContextToolFn, ContextToolFn
-from .tool import AsyncTool, Tool
+from .protocols import AsyncContextToolFn, AsyncToolFn, ContextToolFn, ToolFn
 from .tool_schema import ToolSchema
 
-ContextToolT = TypeVar(
-    "ContextToolT",
+ToolT = TypeVar(
```

Comment:
We never used ToolT, just ContextToolT, so I dropped ContextToolT and changed ToolT's semantics to match. This also aligns with ToolkitT which includes the context variants.